jasmine/jasmine.d.ts

```diff
@@ -382,7 +382,7 @@ declare module jasmine {
         /** By chaining the spy with and.throwError, all calls to the spy will throw the specified value. */
         throwError(msg: string): void;
         /** When a calling strategy is used for a spy, the original stubbing behavior can be returned at any time with and.stub. */
-        stub(): Spy;
+        stub(...params:any[]): Spy;

```

Comment:
hrm... that interpretation (which is what it looks like to me also) seems to contradict the jasmine docs and the test results...? what am I missing I wonder...?